### ISSUE TITLE
Weapons do not place area templates

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#67]
In GitLab by @joshjlewis93 on Sep 5, 2020, 24:17

Spells and feats behave as expected, where templates get automatically placed if they have an area of effect in their target fields (sphere, line etc). Weapons do not behave this way, and chat cards produced for them do not have a button to do it manually afterwards.

Tested with and without Better Rolls (Place Template property enabled).

**Comment by tposney** [comment]
How are you specifying a template for a weapon? On my weapon card there is no option to specify a template for a weapon. If you could attach the json for a weapon I would be grateful.

**Comment by tposney** [comment]
In GitLab by @joshjlewis93 on Sep 5, 2020, 24:35

If it's come from another module, I couldn't say which one as I'm just using the normal item sheet. Here's an example.

[fvtt-Item-Rune__Edge___Ignite.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/3d61aeccc2c2bb18dbbc237938288bc3293d5b7f61c647cfb129046c6e5d9cd9/fvtt-Item-Rune__Edge___Ignite.json)

**Comment by tposney** [comment]
Thanks that was helpful. I've tried out the item and have a few comments. 

When better rolls is enabled I can reproduce your issue. However I did not write better rolls and can't answer for how it decides to place the template or not. I'm not being difficult, I just don't know the code that well. In fact better rolls replaces the whole dnd5e rolling setup. It might be worth contacting @RedReign to see if he can implement templates for weapons.

midi-qol does not do any rolling/template placement, but uses the dnd5e system for that. If better rolls is disabled this is what I get from midi-qol  
![rune_edge](https://my-gitlab-bucket.s3.amazonaws.com/640711674/39ed5ae2108524cdcfb5bd645e7d1c2a349611d38cbc72f97d44172439644d13/rune_edge.png)  
but I note that the auto placement of the template does not happen, nor does the item consumption work properly. In midi-qol I don't actually do the item rolling, but call the dnd5e item.roll(), so the fact that the template is not being placed is a dnd5e system issue. Since I didn't write the dnd5e system it's not something I'm inclined to change.

Having looked at the item, it really should be a feat rather than a weapon and all of the problems will go away.
